radis/api/kuruczapi.py

```diff
@@ -0,0 +1,466 @@
+        nu_lines = 1e8 / wlaa[::-1]  # [cm-1]<-[AA]
+        loggf = loggf[::-1]
+        ielem = ielem[::-1]
+        iion = iion[::-1]
+        elower = elower[::-1]
+        eupper = eupper[::-1]
+        jlower = jlower[::-1]
+        jupper = jupper[::-1]
+        gupper = jupper * 2 + 1
+        A = (
+            10**loggf
+            / gupper
+            * (self.c_CGS * nu_lines) ** 2
+            * (8 * np.pi**2 * self.ecgs**2)
+            / (self.mecgs * self.c_CGS**3)
+        )
+        gamRad = gamRad[::-1]
+        gamSta = gamSta[::-1]
+        gamvdW = gamvdW[::-1]
+
+        data_dict = {
+            "A": A,
+            "nu_lines": nu_lines,
+            "wav": 10**7 / nu_lines,
+            "El": elower,
+            "eupper": eupper,
+            "gu": gupper,
+            "jlower": jlower,
+            "ju": jupper,
+            "id": ielem,
+            "iso": iion,
+            "gamRad": gamRad,
+            "gamSta": gamSta,
+            "gamvdW": gamvdW,
+            "Tdpair": 0.68,  # placeholder to adjust
+            "wlnmair": wlnmair,
+            "loggf": loggf,
+            "species": species,
+            "labellower": labellower,
+            "labelupper": labelupper,
+            "ref": ref,
+            "NLTElower": NLTElower,
+            "NLTEupper": NLTEupper,
+            "isonum": isonum,
+            "hyperfrac": hyperfrac,
+            "isonumdi": isonumdi,
+            "isofrac": isofrac,
+            "hypershiftlower": hypershiftlower,
+            "hypershiftupper": hypershiftupper,
+            "hyperFlower": hyperFlower,
+            "hypernotelower": hypernotelower,
+            "hyperFupper": hyperFupper,
+            "hypternoteupper": hypternoteupper,
+            "strenclass": strenclass,
+            "auto": auto,
+            "landeglower": landeglower,
+            "landegupper": landegupper,
+            "isoshiftmA": isoshiftmA,
+            # "int":self.Sij0(A,gupper,nu_lines,elower,self.partfcn(self.key,296))
+            # if int parameter is used, calc_linestrength_eq will also use it
+        }
+
+        self.data = pd.DataFrame(data_dict)
+        return self.data
+
+    def add_airbrd(self, data):
+        if "airbrd" not in data.columns:
+            # placeholder for neutral_hydrogen_number and atomic_coeff
+            # TODO: adjust the coefficient to the atoms and adjust the value for neutral_hydrogen_number
+
+            neutral_hydrogen_number = 1
+            atomic_coeff = 1
+
+            if self.element_symbol=="H" :
+                airbrd=(10**data['gamvdW'])*data["Tdpair"]*neutral_hydrogen_number
+            elif self.element_symbol== "He":
+                airbrd=(10**data['gamvdW'])*data["Tdpair"]*neutral_hydrogen_number*0.42
+            else :
+                airbrd=(10**data['gamvdW'])*data["Tdpair"]*neutral_hydrogen_number*atomic_coeff
+
+            data["airbrd"] = airbrd
+
+    def store_hdf5(self, data, output_path):
```

Comment:
This should use our Radis DatabaseManager class, which is compatible with Pandas Hdf5 but also other formats such as Vaex Hdf5 or Apache Arrow